gateway/modules/filestore/gcpstorage/delete.go

```diff
@@ -10,17 +10,17 @@ import (
 
 // DeleteFile deletes a file from GCPStorage
 func (g *GCPStorage) DeleteFile(path string) error {
-	// start of path must have single backslash
-	// if end is a backslash, it will delete a folder
-	if strings.HasSuffix(path, "/") {
-		return g.DeleteDir(path)
-	}
+	// trim / at the start and at the end
+	path = strings.TrimPrefix(strings.TrimSuffix(path, "/"), "/")
```

Comment:
Make sure we trim suffix too

Reply:
we are doing it. Look at the first param of trim prefix